launch.go

```diff
@@ -116,19 +116,25 @@ func getSurveyDataHandler(w http.ResponseWriter, r *http.Request) {
 func getSupplementaryDataHandler(w http.ResponseWriter, r *http.Request) {
 	surveyId := r.URL.Query().Get("survey_id")
 	periodId := r.URL.Query().Get("period_id")
-
-	datasets, err := surveys.GetSupplementaryDataSets(surveyId, periodId)
-	if err != nil {
-		http.Error(w, fmt.Sprintf("GetSupplementaryDataSets err: %v", err), 500)
+	sdsEnabled := settings.Get("SDS_ENABLED_IN_ENV")
+
+	if sdsEnabled == "true" {
+		datasets, err := surveys.GetSupplementaryDataSets(surveyId, periodId)
+		if err != nil {
+			http.Error(w, fmt.Sprintf("GetSupplementaryDataSets err: %v", err), 500)
+			return
+		}
+		datasetJSON, _ := json.Marshal(datasets)
+
+		_, writeError := w.Write([]byte(datasetJSON))
+		if writeError != nil {
+			http.Error(w, fmt.Sprintf("Write failed to write data as part of an HTTP reply: %v", writeError), 500)
+			return
+		}
+	} else {
```

Comment:
Minor - could maybe flip the logic here into a guard condition, so that we return from the function a bit earlier on if `sdsEnabled` is false?

Reply:
I've changed this to add a guard condition but because of the way its structured it should be exiting the function at a similar point. Have a look and let me know if this is what you meant.

Reply:
Looks good! 👍 Yep that's exactly what I had in mind